app/gui2/e2e/actions.ts

```diff
@@ -57,7 +57,13 @@ export async function dragNodeByBinding(page: Page, nodeBinding: string, x: numb
 }
 
 /// Move mouse away to avoid random hover events and wait for any circular menus to disappear.
-export async function ensureNoCircularMenusVisible(page: Page) {
+export async function ensureNoCircularMenusHovered(page: Page) {
```

Comment:
Wasn't old name actually better? In this function, we expect `circularMenu` is indeed not visible.